### ISSUE TITLE
Another 403 Client Error: Forbidden for url

### ISSUE THREAD

**Issue by thewracl** [rarcega/instagram-scraper#316]
Again I got the error

`2018-11-30 20:57:18,031 - instagram_scraper.app - WARNING - Retry after exception HTTPError(u'403 Client Error: Forbidden for url: https://scontent-frt3-2.cdninstagram.com/vp/26f867e1414c9aab720210f276c235d3/5C03CFC9/t51.12442-15/e15/46649260_1142232549263250_8339527832475516529_n.jpg',) on https://scontent-frt3-2.cdninstagram.com/vp/26f867e1414c9aab720210f276c235d3/5C03CFC9/t51.12442-15/e15/46649260_1142232549263250_8339527832475516529_n.jpg`

It stays again on an endless loop when using the forever option

Looks like Instagram changed something again

EDIT: This happened while scraping 'supervandie' but I also have other (private) profiles where this happens. From different user accounts ...

**Comment by clublek** [comment]
Can confirm i have the same errors since yesterday...

**Comment by thewracl** [comment]
@VasiliPupkin256 and/or @pr1m3r 
If one of you is still active, can you please look at that problem? Many thanks! I can provide further data if you say what you need :-)

**Comment by TarzanBuzzyMan** [comment]
I think the issue lies with the complete url requiring "?_nc_ht=instagram.fakl1-1.fna.fbcdn.net" to be added to the end of it sometimes.

So in the mean time if it errors, just append this string to the end of the url that is returned as part of the error in the command line and see if that gives you valid media in a web browser. 

?_nc_ht=instagram.fakl1-1.fna.fbcdn.net

Also FYI adding this to every url string will make most of them error and will only work for ones requiring it. So don't go adding it to every url yet!

**Comment by VasiliPupkin256** [comment]
Yep, just remove this split function call there on the line 845, the `download` function already has the functionality to strip url from parameters and retry with the full url if got 403 error

**Comment by ano0** [comment]
works fine after removing that function

**Comment by thewracl** [comment]
Sorry for my late answer. I removed the line and tested it further. At first everything seems OK, it continues  further but then I noticed that I no longer get preview pictures from the story movies. Maybe these where the jpg files that made trouble all along. Can you verify this?

**Comment by clublek** [comment]
Can confirm this, video's are now missing the separate video thumbnail download.

**Comment by pr1m3r** [comment]
can't confirm but still using slightly changed version of 1.5.39 and only changed split function to use '&' as seperator. you could try using this seperator as well and report impressions

**Comment by VasiliPupkin256** [comment]
Why have you closed it? It is not fixed in the code

**Comment by ekalin** [comment]
I've tested simply removing the `split` (ie, using the URL as it comes from the server) and it seems to work.

In practice this is the same as splitting on `&` as currently there is no `&` in the URLs, but someday they might include more query parameters, and it might stop working if they are not all included.

For me the change did not cause the program to stop downloading stories or the images of story videos. I don't see how such a small code change would affect that; it's probably an unrelated problem (Instagram is always changing its API...)

A pull request would be nice, it's better if the original author of the solution does that so he can get the credit. :-)
